src/tests/ftest/util/AgentUtils.py

```diff
@@ -118,4 +171,4 @@ def stop_agent(client_list):
     # this kills the agent
     for client in client_list:
         cmd = "pkill daos_agent"
-        resp = subprocess.call(["ssh", client, cmd])
\ No newline at end of file
+        dummy_resp = subprocess.call(["ssh", client, cmd])
```

Comment:
Is `dummy_resp` used anywhere?  If not how about just:

```suggestion
        subprocess.call(["ssh", client, cmd])
```

But in either case, what happens if this `ssh` fails and the agent is not killed (perhaps for a different reason even)?  Certainly for the case of where it's just being stopped during end of test cleanup, the agent not stopping is just annoying, but this function could be called in the middle of a test run where the agent failing to stop to could follow-on failures.